inline.h

```diff
@@ -3698,6 +3740,46 @@ Perl_my_strlcpy(char *dst, const char *src, Size_t size)
 }
 #endif
 
+/*
+
+=for apidoc sv_nosharing
+
+Dummy routine which "shares" an SV when there is no sharing module present.
+Or "locks" it.  Or "unlocks" it.  In other
+words, ignores its single SV argument.
+Exists to avoid test for a C<NULL> function pointer and because it could
+potentially warn under some level of strict-ness.
+
+=cut
+*/
+
+PERL_STATIC_INLINE void
+Perl_sv_nosharing(pTHX_ SV *sv)
+{
+    PERL_UNUSED_CONTEXT;
+    PERL_UNUSED_ARG(sv);
+}
+
+/*
+
+=for apidoc sv_destroyable
+
+Dummy routine which reports that object can be destroyed when there is no
+sharing module present.  It ignores its single SV argument, and returns
+'true'.  Exists to avoid test for a C<NULL> function pointer and because it
+could potentially warn under some level of strict-ness.
+
+=cut
+*/
+
+PERL_STATIC_INLINE bool
+Perl_sv_destroyable(pTHX_ SV *sv)
+{
+    PERL_UNUSED_CONTEXT;
+    PERL_UNUSED_ARG(sv);
+    return TRUE;
+}
+
```

Comment:
These are only called via function pointers, there's no point to inlining them

Reply:
PR updated to not include these two.